core/pom.xml

```diff
@@ -54,6 +54,13 @@
 			<version>1.4.11</version>
 			<scope>compile</scope>
 		</dependency>
+		<dependency>
+			<groupId>redis.clients</groupId>
+			<artifactId>jedis</artifactId>
+			<version>2.9.0</version>
+			<scope>compile</scope>
```

Comment:
Should probably be "optional", and we should change it for spymemcached etc probably as well.